Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -435,6 +435,8 @@ extension TelemetryWrapper {
         case mediumTabsOpenUrl = "medium-tabs-widget-url"
         case largeTabsOpenUrl = "large-tabs-widget-url"
         case smallQuickActionSearch = "small-quick-action-search"
+        case smallQuickActionClosePrivate = "small-quick-action-close-private"
+        case smallQuickActionCopiedLink = "small-quick-action-copied-link"
```

Comment:
What was sent before if we didn't have those?

Reply:
Telemetry for the `medium widget` was sent which I think was an oversight. You can see it [here](https://github.com/mozilla-mobile/firefox-ios/blob/ed54544fe1676dfa2e88c2d04146237c8b3ae1a0/Client/Application/NavigationRouter.swift#L230)